clients/web/src/views/widgets/HierarchyWidget.js

```diff
@@ -117,6 +118,18 @@ var HierarchyWidget = View.extend({
 
         this._showActions = _.has(settings, 'showActions') ? settings.showActions : true;
         this._showItems = _.has(settings, 'showItems') ? settings.showItems : true;
+
+        /*
+         * require that selectFolder be set *and* that a folder select callback
+         * be provided
```

Comment:
Good point.  I'm trying to remember what my thought process was here, but I think I agree with you.  This part also reminds me of a "foldersOnly" option I meant to add; where the view would only show folders (and not items).  I wonder if maybe I somehow got that mixed up here?